### PR TITLE
Google Sign In Button: Functionality 

### PR DESCRIPTION
**Pull request by ChineyeEmma**
The request is regarding the addition of the Google Sign In button. I have added the functionality of the button, meaning I am able to display and retrieve the user's name, email, and profile photo.

### CODE REVIEW

File: src/main/java/codeu/controller/ChatServlet.java [hunk 1]
```diff
@@ -113,7 +113,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     * the logged-in username from the session, the conversation title from the URL,
     * and the chat message from the submitted form data. It creates a new Message
     * from that data, adds it to the model, and then redirects back to the chat
-    * page.
+    * page.q
```
Comment (Dash433) [inline src/main/java/codeu/controller/ChatServlet.java]: I do not think we need this q right here

File: src/main/webapp/WEB-INF/view/script.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+function onSignIn(googleUser){
+
+	var profile=googleUser.getBasicProfile();
+	$(".g-signin").css("display","none");
+	$(".data").css("display","block");
+}
```
Comment (Dash433) [inline src/main/webapp/WEB-INF/view/script.js]: Try to add all of the javascript code to the js file

File: src/main/webapp/script.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+{
+	var auth2 = gapi.auth2.getAuthInstance();
+	auth2.signOut().then(function()){
+
+		alert("You have been signed out, have a good one!");
+
+		$(".g-signin2").css("display", "none");
+
+	};
+}
```
Comment (Dash433) [inline src/main/webapp/script.js]: again add this to the js file

File: src/main/webapp/WEB-INF/view/login.jsp [hunk 4]
```diff
@@ -48,7 +73,23 @@
...
+          <!--Profile Pic-->
+          <img id="pic" class="img-circle" width="100" height="100"/>
+          <!--Email Address-->
+          <p>EMAIL</p>
+          <p id="email" class= "alert-danger"></p>
+         
+        </div>
+
+
     </form>
```
Comment (Dash433) [inline src/main/webapp/WEB-INF/view/login.jsp]: Really great work right here. Nice job describing what its supposed to look like

File: src/main/webapp/WEB-INF/view/script.js (hunk 2 above)
Comment (jonathantanmy) [reply src/main/webapp/WEB-INF/view/script.js]: John: You'll need to add more context. Here, this JavaScript code is already in a js file.

John probably means that all the JavaScript code should go in one file, and I agree - I suggested src/main/webapp/js/script.js, but if you decide on something else, that's fine too.

**Pull request merged**